Monika After Story/game/script-topics.rpy

```diff
@@ -15131,16 +15131,29 @@ init 5 python:
     )
 
 label monika_dying_same_day:
-    m 1euc "[player], have you ever heard of couples wishing to die on the same day?"
-    m 3rksdla "...Gosh, that sounds kinda dark, doesn't it?"
-    m 3rfd "Before I met you, I thought the idea was completely absurd!"
-    m 1etc "Why would someone give up on living a long and happy life just because their partner's life was dramatically cut short?"
-    m 3rfd "It would be irrational and impulsive to do so!"
-
-    if mas_isMoniEnamored(higher=True):
-        m 2ekc "...But with you by my side and all the time I've spent here with you...{w=0.3}I can kind of see it now."
+    # NOTE: Dummy value.
```

Comment:
We have `mas_event_mailbox` to check who started the topic and how. In your current code you could use `mas_event_mailbox.is_player_initiated_mode()`.

But I think it's better to check for `mas_event_mailbox.is_monika_initiated_mode()` instead because only in this one she'd try to get your attention, really. So you'll need swap the `if` and `else`